packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -143,8 +143,9 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
   let compiler = await createCompiler(input)
   let scanner = new Scanner({
     detectSources: { base },
-    sources: compiler.globs.map((pattern) => ({
-      base: inputBasePath, // Globs are relative to the input.css file
+    sources: compiler.globs.map(({ origin, pattern }) => ({
+      // Globs are relative to the input.css file
```

Comment:
I know this comment was already there but I wonder if it's misleading to say `input.css` here since the file could be named anything? And `origin` comes from a config file, not a CSS file.

Maybe these should be more like:

> Globs are relative to the CSS file or config file where they are specified

Reply:
I left it there because the `origin` is also relative to the input CSS file right now, but yeah this is confusing. Maybe:

> Globs need to be relative to the CSS file or config file where they are specified. An eventual origin is relative to the CSS file.

But this is really confusing now 😬

Reply:
How about:

> Ensure the glob is relative to the input CSS file or the config file where they are specified.

Reply:
Updated it to the latest one